Backend/Gmfctn/Controllers/AchievementController.cs

```diff
@@ -49,7 +52,63 @@ public ActionResult<Achievement> GetAchivementById(int id)
             {
                 throw exc;
             }
-
+        }
+        [HttpPost]
+        public async Task<ActionResult> CreateAchievement(AchievementCreateDTO element)
+        {
+            try
+            {
+                var achievement = _mapper.Map<Achievement>(element);
+                achievement.Id = new Guid();
+                await unitOfWork.AchievementRepository.Create(achievement);
+                await unitOfWork.AchievementRepository.SaveChanges();
```

Comment:
You are using here 'await' but your method is not Async. So keep in mind that you need to use 'await' only with async methods.